lms/static/sass/course/_profile.scss

```diff
@@ -187,14 +187,16 @@
       > .credit-eligibility-container {
           padding: lh();
         > .credit-help {
+          display: inline-block;
           background: $blue;
           color: $white;
-          width: lh();
+          width: lh(0.6);
           margin: 0;
           padding: 0;
           border-radius: lh(0.9);
           border-color: $white;
           text-shadow: None;
+          padding-left: 0.2em;
         }
```

Comment:
Please add RTL support (see this file for mixins https://github.com/edx/edx-platform/blob/master/lms/static/sass/vendor/bi-app/_mixins.scss)